src/Provider.js

```diff
@@ -1,16 +1,16 @@
-import React, { Component, PropTypes as T, Children } from 'react';
-import { AlgoliaSearchHelper } from 'algoliasearch-helper';
+import {Component, PropTypes, Children} from 'react';
 
-import createStore from './createStore';
-import storeShape from './storeShape';
+import createStore from './createStore.js';
+import storeShape from './storeShape.js';
```

Comment:
So that I can infer from the code that I am requiring a file and not potentially requiring storeShape/index.js. It's a preference mostly.